config/base.yaml

```diff
@@ -112,6 +112,48 @@ bert:
     --model: "Bert"
     --batch-size: 32
 
+fp32:
+  inherits: _hf
+  tags:
+    - showcase
+  argv:
+    --model: "Bert"
+    --fixed-batch: true
```

Comment:
That argument doesn't actually do anything anymore. It's always a fixed batch. I forgot to remove it.

Reply:
done